stdlib/2/os/__init__.pyi

```diff
@@ -94,10 +95,58 @@ WUNTRACED = 0  # Unix only
 
 TMP_MAX = 0  # Undocumented, but used by tempfile
 _PathType = Union[bytes, Text]
-_StatVFS = NamedTuple('_StatVFS', [('f_bsize', int), ('f_frsize', int), ('f_blocks', int),
-                                   ('f_bfree', int), ('f_bavail', int), ('f_files', int),
-                                   ('f_ffree', int), ('f_favail', int), ('f_flag', int),
-                                   ('f_namemax', int)])
+
+class stat_result(NamedTuple('stat_result', [
+        ('st_mode', int),  # protection bits,
+        ('st_ino', int),  # inode number,
+        ('st_dev', int),  # device,
+        ('st_nlink', int),  # number of hard links,
+        ('st_uid', int),  # user id of owner,
+        ('st_gid', int),  # group id of owner,
+        ('st_size', int),  # size of file, in bytes,
+        ('st_atime', float),  # time of most recent access,
+        ('st_mtime', float),  # time of most recent content modification,
+        ('st_ctime', float)])):  # platform dependent (time of most recent metadata change on Unix, or the time of creation on Windows)
+
+    # For backward compatibility, the return value of stat() is also
+    # accessible as a tuple of at least 10 integers giving the most important
+    # (and portable) members of the stat structure, in the order st_mode,
+    # st_ino, st_dev, st_nlink, st_uid, st_gid, st_size, st_atime, st_mtime,
+    # st_ctime. More items may be added at the end by some implementations.
+
+    if sys.version_info >= (3, 3):
+        st_atime_ns = 0  # time of most recent access, in nanoseconds
```

Comment:
Better to write `st_atime_ns: int` (or `st_atime_ns = ...  # type: int`, but I think we've established that all users of typeshed obey PEP 526 now).